src/modules/roc_sndio/target_sox/roc_sndio/player.h

```diff
@@ -60,11 +58,23 @@ class Player : public core::Thread {
     //!  Should be called once before calling start().
     bool open(const char* name, const char* type = NULL);
 
+    //! Start reading samples.
```

Comment:
It's worth to mention that it starts a thread.

Reply:
Fixed